### PR TITLE
Update Python_CheatSheet.md with two new commands

### PR DESCRIPTION
**Pull request by prabhav5112**

### CODE REVIEW

File: Python_CheatSheet.md [hunk 1]
```diff
@@ -25,4 +25,9 @@
 	#Redirect the modules and dependencies installed to requirements.txt.
 		pip3 freeze > requirements.txt
 
+    #Install modules whose names are listed in a file (requirements.txt)
+        pip3 install -r requirements.txt
+        
+    #Sort a 2D list (lst) based on any one index (i is the index)
+        lst.sort(key = lambda x : x[i])
```
Comment (Jahenr) [inline Python_CheatSheet.md:32]: Same with this one too please.

**Review by Jahenr — CHANGES_REQUESTED** [review]

**Comment by Jahenr** [comment]
Hi prabhav5112

Thank you for the contribution, please review the comments then I can get that merged for you. 👍🏿

**Comment by prabhav5112** [comment]
Resolved the indentation for both commands

**Comment by Jahenr** [comment]
Thank you, it looks good. I appreciate the help, take care 👍🏿

**Pull request merged**